telemetry/roles/idrac_telemetry/tasks/install_check.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Initialise install_idrac_telemetry variable
+  ansible.builtin.set_fact:
+    install_idrac_telemetry: false
+
+- name: Check idrac-telemetry pod status
```

Comment:
Shouldn't this be specific?

Reply:
We are using this task for idrac telemetry pod status and skip installation in 2nd run